js_modules/dagster-ui/packages/ui-core/src/sensors/SensorRoot.tsx

```diff
@@ -230,17 +230,24 @@ export const SENSOR_ASSET_SELECTIONS_QUERY = gql`
 
   fragment SensorAssetSelectionFragment on AssetSelection {
     assetSelectionString
-    assets {
-      id
-      key {
-        path
-      }
-      definition {
-        id
-        autoMaterializePolicy {
-          policyType
+    assetsOrError {
+      ... on AssetConnection {
+        nodes {
+          id
+          key {
+            path
+          }
+          definition {
+            id
+            autoMaterializePolicy {
+              policyType
+            }
+          }
         }
       }
+      ... on PythonError {
+        ...PythonErrorFragment
+      }
```

Comment:
I don't think you need the `on PythonError` since that's specified on the fragment.

```
... on AssetConnection {
  // etc
}
...PythonErrorFragment
```